src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -776,61 +834,12 @@ public void onSctpPacket(
             byte[] data, int sid, int ssn, int tsn, long ppid, int context,
             int flags)
     {
-        if (ppid == WEB_RTC_PPID_CTRL)
-        {
-            // Channel control PPID
-            try
+        sctpDispatcher.execute(() -> {
```

Comment:
Is there a chance for the data to be altered from some other thread? The buffer is passed by reference, so in theory it's possible' but, in practice, that shouldn't happen in our case, right?